.evergreen/auth_aws/activate_venv.sh

```diff
@@ -0,0 +1,25 @@
+set +x
+
+if [ "Windows_NT" = "$OS" ]; then
+  PYTHON_BINARY=python.exe
```

Comment:
In pymongo and .NET on windows we use:
```
  PYTHON_BINARY=C:/python/Python38/python.exe
```

Reply:
More explicit is probably better, but I wanted to use the same logic as [here](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/auth_aws/lib/aws_e2e_lib.js#L33-L39) to make sure they are using the same python; I wonder if it'd make sense to update that in both places?

Reply:
The logic in that file assumes the javascript file is running inside a virtualenv. The logic here to create that virtualenv is different. We have to change this to:
```
PYTHON_BINARY=C:/python/Python38/python.exe
```
Or else we'll end up using Python 2.7 which won't work.

>  I wonder if it'd make sense to update that in both places?

Nope the js file should remain the same. It's a quirk that windows virtualenvs don't create an alias for "python3", only  "python".